gamestonk_terminal/cryptocurrency/binance_model.py

```diff
@@ -0,0 +1,253 @@
+"""Binance model"""
+__docformat__ = "numpy"
+
+import argparse
+from typing import List
+import pandas as pd
+import numpy as np
+from binance.client import Client
+from binance.exceptions import BinanceAPIException
+from gamestonk_terminal.main_helper import parse_known_args_and_warn
+from gamestonk_terminal.config_terminal import BINANCE_API_KEY, BINANCE_SECRET_KEY
+from gamestonk_terminal.cryptocurrency.binance_view import plot_order_book, plot_candles
+
+
+def check_valid_binance_str(symbol: str):
+    """Check if symbol is in defined binance"""
+    client = Client(BINANCE_API_KEY, BINANCE_SECRET_KEY)
+    try:
+        client.get_avg_price(symbol=symbol.upper())
+        return symbol.upper()
+    except BinanceAPIException as e:
+        raise argparse.ArgumentTypeError(
+            f"{symbol} is not a valid binance symbol"
+        ) from e
+
+
+# pylint: disable=inconsistent-return-statements
+
+
+def add_binance_coin(other_args: List[str]):
+    """
```

Comment:
I think this should be:
```
"""Define coin-currency pair from binance

Parameters
```

Note the spacing and that the description starts on the first line.